### PR TITLE
feat(general) : MP-231 Snackbar Layout

### PR DESCRIPTION
**Pull request by RTAgung**
## Update Items
- [x] [[MP-231](https://args06.atlassian.net/browse/MP-231)] Snackbar Layout

## Summary Description
- Create Snackbar Layout
- Add new sample error icon

## *Type of Change
- [ ] docs: Documentation only changes, README.md, Wiki
- [x] feat: A new feature
- [ ] refactor: A code change that neither fixes a bug nor adds a feature
- [ ] test: Anything related to unit / integration testing
- [ ] revert: Revert a commit

## Type of Scope
- [ ] album: Changes that affect user packages
- [ ] music: Changes that affect music packages
- [ ] account: Changes that affect account packages
- [ ] favorite: Changes that affect favorite packages
- [ ] playlist: Changes that affect playlist packages
- [ ] playlist-detail: Changes that affect playlist-detail packages
- [ ] music-artist: Changes that affect music-artist packages
- [x] general: all stuff except above

### CODE REVIEW

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     </style>
 
     <style name="CenturyGothicTextViewStyle" parent="android:Widget.TextView">
-        <item name="android:fontFamily">@font/century_gothic_bold</item>
+        <item name="android:fontFamily">@font/century_gothic</item>
```
Comment (args06) [inline app/src/main/res/values/themes.xml:21]: Ini kenapa diganti ke yg biasa? Dari figma soalnya yang bold

**Review by args06 — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/themes.xml (hunk 1 above)
Comment (RTAgung) [reply app/src/main/res/values/themes.xml:21]: ngaruh nya lebih banyak ketempat lain juga, jadi mending general aja, yg butuh bold nanti tinggal di kasih style

Comment (args06) [reply app/src/main/res/values/themes.xml:21]: Kalo gitu ntar jadi beda sama figma?

Comment (RTAgung) [reply app/src/main/res/values/themes.xml:21]: itu style global kan? banyak jg textview yg ga pake bold

**Review by args06 — APPROVED** [review]

**Pull request merged**